src/Api/Wiki.php

```diff
@@ -17,8 +17,8 @@
 class Wiki extends AbstractApi
 {
     /**
-     * @param int|string $project_id
-     * @param array      $params
+     * @param int|string          $project_id
+     * @param array<string,mixed> $params
```

Comment:
Please adjust this doc to be similar to the other parameters doc, and use the options resolver.

Reply:
Done.